src/silx/gui/data/DataViews.py

```diff
@@ -1066,12 +1069,35 @@ def createWidget(self, parent):
         widget.setDefaultColormap(self.defaultColormap())
         widget.getColormapAction().setColormapDialog(self.defaultColorDialog())
         widget.getIntensityHistogramAction().setVisible(True)
+
+        self.__aggregationModeAction = AggregationModeAction(parent=widget)
+        widget.toolBar().addAction(self.__aggregationModeAction)
+        self.__aggregationModeAction.setVisible(True)
+        self.__aggregationModeAction.sigAggregationModeChanged.connect(self._aggregationModeChanged)
+
         widget.setKeepDataAspectRatio(True)
         widget.getXAxis().setLabel("X")
         widget.getYAxis().setLabel("Y")
         maskToolsWidget = widget.getMaskToolsDockWidget().widget()
         maskToolsWidget.setItemMaskUpdated(True)
         return widget
+    
+    def getAggregationModeAction(self):
+        """Action toggling the aggregation mode action
+
+        :rtype: actions.PlotAction
+        """
+        return self.__aggregationModeAction
```

Comment:
It's nowadays best to use typing rather than `:rtype:` in docstring since the type serves 2 purposes: it is used to generate the documentation and also slowly starts to type the project's API:

```suggestion
    def getAggregationModeAction(self) -> AggregationModeAction:
        """Action toggling the aggregation mode action"""
        return self.__aggregationModeAction
```